pkg/lib/machine-info.jsx

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
Nitpick: this isn't really JS"X", so just .js (or  maybe .es6 if you want to use some more modern features ☺) seems more appropriate. Not important, though.

Reply:
Ahh, I wanted es6, but didn't know about the `.es6` suffix.  Great!